server/index.js

```diff
@@ -1,3 +1,6 @@
-const server = require('flatlify-server');
+const { bootstrap } = require('@flatlify/flatlify');
+const dotenv = require('dotenv');
```

Comment:
move `dotenv.config();` before 
`const { bootstrap } = require('@flatlify/flatlify');`

just to be safe